components/x-interaction/src/Hydrate.jsx

```diff
@@ -52,6 +52,12 @@ export function hydrate() {
 
 		const Component = getComponent(component)
 
+		if (!Component) {
+			throw new Error(
+				`x-interaction hydrate was called using unregistered component: ${Component}. please verify you'are registering your component using x-interaction's registerComponent function before attempting to hydrate.`
```

Comment:
this should log `component` (i.e. the name of the component), not `Component` which is going to be `undefined` here